### ISSUE TITLE
False positive, "Method "line_exists" not found"

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1205]
Reproducible with:

```abap
DATA t_data TYPE STANDARD TABLE OF i.
IF line_exists( t_data[ table_line = 2 ] ).
ENDIF.
```

And in https://playground.abaplint.org

![image](https://user-images.githubusercontent.com/5888506/90773098-ba60ea80-e2f5-11ea-8632-7d39e081c384.png)
